client/my-sites/stats/stats-period-header/style.scss

```diff
@@ -35,3 +36,19 @@
 		margin-top: 0;
 	}
 }
+
+// styling for both interval dropdown and date control picker drop down buttons inside the header
+.stats-date-control {
+	.components-dropdown > .components-button {
+		--gray-gray-10: #c3c4c7;
+		--gutenberg-gray-900: #1e1e1e;
+		background: #fff;
+		border-radius: 4px;
+		border: 1px solid var(--gray-gray-10);
+		color: var(--studio-black);
+		font-family: $font-sf-pro-display;
+		font-size: 14px; /* stylelint-disable-line declaration-property-unit-allowed-list */
+		font-weight: 500;
+	}
+}
+
```

Comment:
The comment notes this is for both buttons in the date control but currently it only applies to one of them. Only the interval button is inside the `components-dropdown` DIV.

It might be enough to just remove the `.components-dropdown > ` bit here. If not, I'd assign a new class to those top-level buttons and target them directly.

![SCR-20231004-ldkk](https://github.com/Automattic/wp-calypso/assets/40267301/9065cfaf-d2d5-4190-9243-25cffcdeb558)

Reply:
Yep this is in the plans - the main reason being that to get the layout we want, I am pretty sure we area going to need to relocate the interval button outside of the current containing component. So I didn't go too deep into the structure here and will fix this up in the next PR :)